webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -124,7 +124,7 @@ export default class CreateIssueModal extends PureComponent {
         if (error) {
             submitError = (
                 <p className='help-text error-text'>
-                    <span>{error}</span>
+                    <span>{JSON.parse(error).message}</span>
```

Comment:
I'm concerned that `error` may sometimes be a string here. Maybe we should protect against that?

Reply:
If it is the case that `error` is a string, then this will error out and cause WSOD. I suggest applying a fix in this component's [handleCreate](https://github.com/mattermost/mattermost-plugin-github/blob/ad7f5ba777f96f980f8be27dad143518eabb70d8/webapp/src/components/modals/create_issue/create_issue.jsx#L69) method instead of the render function, similar to the fix in the Attach Comment to Issue Modal:

https://github.com/mattermost/mattermost-plugin-github/blob/bc5a37107122d81aff17b2a44fdadd28b89403e3/webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx#L61-L66

Reply:
@aidapira What are your thoughts on moving the fix out of the `render` function and into `handleCreate`?

Reply:
Hi @mickmister, that sounds like a great idea. Please give me a few days and I will commit the updates based on your comment.